powershell/cmdlets/class.ts

```diff
@@ -1731,3 +1732,12 @@ export class CmdletClass extends Class {
     }
   }
 }
+function GetParameterReferenceString(typeName: string): string {
+  if (typeName.endsWith("[]")) {
+    return `<see cref="${typeName.substring(0, typeName.lastIndexOf("[]"))}">${typeName}</see>`;
+  } else if (typeName.endsWith("?")) {
+    return `<see cref="${typeName.substring(0, typeName.lastIndexOf("?"))}">${typeName}</see>`;
+  }
+  return `<see cref="${typeName}"/>`;
+}
+
```

Comment:
Is there any advice? I prefer regex here. But I'm unfamiliar with typescript.

Reply:
How about to replace the GetParameterReferenceString function with following one line code.

<see cref="${each.language.csharp.responseType.replace(/\[|\]|\?/g, '')}">${each.language.csharp.responseType}</see>

Reply:
Should be 
```
<see cref="${each.language.csharp.responseType.replace(/\[|\]|\?/g, '')}">${each.language.csharp.responseType}</see>
```

Reply:
Thank you!